IntelFsp2Pkg/FspSecCore/X64/FspApiEntryCommon.nasm

```diff
@@ -52,7 +52,9 @@ FspApiCommon1:
   PUSHA_64
   mov    rdx, rcx           ; move ApiParam to rdx
   mov    rcx, rax           ; move ApiIdx to rcx
+  sub    rsp, 0x20

```

Comment:
64bit MSABI calling conversion requires the RSP is 16-byte aligned before "call".

Here we are calling FspApiCallCheck with 8-byte aligned stack.